modules/salt/pages/config-modules.adoc

```diff
@@ -0,0 +1,41 @@
+[[config-modules]]
+= Configuration Modules
+
+[IMPORTANT]
+====
+This feature is a technology preview.
+====
+
+Salt execution and state modules are used to configure your installation.
```

Comment:
```suggestion
Execution and state modules are the main concepts in Salt to define, apply and orchestrate configuration of your devices.
```

Reply:
Slightly edited this to make it clearer, what do you think?

Reply:
:+1: